oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/cache/NetworkAddressAliasEsDAO.java

```diff
@@ -62,8 +64,10 @@ public List<NetworkAddressAlias> loadLastUpdate(long timeBucketInMinute) {
 
             SearchResponse results =
                 getClient().search(NetworkAddressAlias.INDEX_NAME, search, params);
-            final String scrollId = results.getScrollId();
+            final Set<String> scrollIds = new HashSet<>();
```

Comment:
As the scroll IDs are changing, is a list enough for this?

Reply:
> As the scroll IDs are changing, is a list enough for this?

It's "sometimes changed" 🤦🏻, see the link I posted in comment